### PR TITLE
Button styles, new favicon, and beginnings of the easter egg 

### PR DESCRIPTION
**Pull request by brunnerjosh**

### CODE REVIEW

File: packages/react-scripts/template/src/Styles/Button.styl [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  font-size: 1em
+  color: $primary
+  cursor: pointer
+  padding: .5em 1em
+  background: $white
+  border-radius: rem(5);
+  transition: background 250ms ease
+
+  &:hover
+    background: darken($white, 15)
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Button.styl]: Move to Landing.styl

File: packages/react-scripts/template/src/Styles/Landing.styl [hunk 2]
```diff
@@ -34,3 +38,51 @@
...
+  60% {
+    opacity: 0.6;
+  }
+  80% {
+    opacity: 0.8;
+  }
+  100% {
+    opacity: 1;
+  }
+}
```
Comment (canfie1d) [inline packages/react-scripts/template/src/Styles/Landing.styl:100]: Only need 0% and 100%

**Review by canfie1d — CHANGES_REQUESTED** [review]
Just 2 comments

File: packages/react-scripts/template/src/Styles/Landing.styl (hunk 2 above)
Comment (brunnerjosh) [reply packages/react-scripts/template/src/Styles/Landing.styl:100]: This is removed in the easter egg PR and can be ignored.

**Review by canfie1d — APPROVED** [review]

**Pull request merged**